app/src/main/res/layout/custom_list.xml

```diff
@@ -41,7 +41,7 @@
             android:paddingTop="20sp"
             android:text="Days"
             android:textAlignment="textStart"
-            android:textColor="#3F51B5" />
+            android:textColor="@color/textViewColor" />
```

Comment:
I created a constant for textviewcolors. That way it can be kept constant over your project, instead of having to using hexadecimals. If need to change color in the future, you can change it in the color.xml file.

Reply:
Good.